address/static/address/css/address.css

```diff
@@ -0,0 +1,3 @@
+.pac-container {
+    z-index: 1051 !important;
```

Comment:
Is this directly relevant to `django-address`? It seems like this is something that should belong in a CSS file specific to your site.